### ISSUE TITLE
Improve the doc regarding the content of bootstrap.yaml file

### ISSUE THREAD

**Issue by charlesprost** [scality/metalk8s#1218]
**Component**:

documentation

**Why this is needed**:

Because we are tempted to edit manually some parts of this file when we should not, and because some parts are unclear.

**What should be done**:

* ~~remove the harcoded "v1alpha2" version from the doc and from the test running during the execution.~~
* specify that running the `hostname` command to get the true hostname is necessary
* specify the IP address we want for apiServer is the one from the control plane network

**Implementation proposal** (strongly recommended):

* edit documentation
* ~~remove the line that tests the version from `salt/_pillar/metalk8s.py`~~

**Test plan**:

* bootstrap a node with another version and see if it works

**Comment by gdemonet** [comment]
> remove the harcoded "v1alpha2" version from the doc and from the test running during the execution

No. That's the format version of the file, which will change with time. Although we could explain what it is in the docs, it's quite standard in the K8s world (users don't change the `version` when provided `Pod`/`Deployment` objects, for instance).

> specify the IP address we want for apiServer is the one from the control plane network

This part also needs to be extended - we also support deploying `keepalived` for HA of k-a, which is configured in the same `apiServer` section.

**Comment by charlesprost** [comment]
OK for the format version, I'll edit the ticket description.

And agreed for the keepalived stuff. But that's another story I guess.
